.env

```diff
@@ -40,3 +40,4 @@ DAI_ADDRESS=0x4F96Fe3b7A6Cf9725f59d353F723c1bDb64CA6Aa
 USDC_ADDRESS=0x75B0622Cec14130172EaE9Cf166B92E5C112FaFF
 POOLTOGETHER_GRAPH_ID=QmSmtdcYD9Zuk7W9zuk73rRCNema26vMdF7muyfGwLzqfT
 THE_GRAPH_API_URL=https://api.thegraph.com/subgraphs/id
+SHOW_LANG_TOGGLE=
```

Comment:
I'm assuming this is a boolean value, should we put at least _something_ here?

Reply:
ENV variables are treated like strings, so if we set it to `false` here, we'll get it as `'false'` which ends up to be truthy value.
I'm leaving it empty so it would be `undefined`. 
(And I'm leaving it empty on dev as well, as it is not always needed - just when we would be working on translations)

Reply:
Ahh forgot about the ENV strings!